### PR TITLE
Show followers you know

### PR DESCRIPTION
**Pull request by Hermanya**
image

What inspired me to make this, is the twitter's profile page:

image

The reason I think this is important is same as #1392. Social proof is useful for discovering people. This updated profile page can show how you are connected to a particular person. If you have friends in common, then that friend might be able to introduce you.
Or, if you don't know why this profile is worth following, the person in common can explain.

"Who to follow" is another feature I like on twitter. I was thinking how I could make it work for github, but I have not found a way just yet. Maybe it's the people, who the person you just followed, follows. It's very useful for people discovery 👍

### CODE REVIEW

**Comment by sindresorhus** [comment]
This is a great idea!

**Comment by sindresorhus** [comment]
Can you use the GitHub tooltips when hovering over the avatars? Should show the username and if available the full name.

Like: 
screen shot 2018-07-11 at 04 39 41

File: source/features/show-followers-you-know.js [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+	return stargazers;
+};
+
+const avatarHeight = 35;
+const renderAvatar = ({link, description, avatar}) => (
+	<a href={link} title={description} class="avatar-group-item mr-1">
+		<img
+			class="avatar"
+			src={`${avatar}?s=${avatarHeight * window.devicePixelRatio}`}
+			alt={description} height={avatarHeight} />
```
Comment (sindresorhus) [inline source/features/show-followers-you-know.js]: Set the `width` and `height` attributes.

**Comment by sindresorhus** [comment]
Would be useful to also show whether the user is following you.

**Comment by Hermanya** [comment]
Thanks for the feedback.  

From what I understand, refined github is focused on features for power users. When I went through the list on your readme, I realized that I didn't care for most of them. As I'm not a power user I guess. 

I forked refined github (https://github.com/hermanya/friendly-github) and am planning to add similar features in there.

**Comment by sindresorhus** [comment]
> I forked refined github (hermanya/friendly-github) and am planning to add similar features in there.

Does that mean you're not interested in finishing this PR?

**Comment by sindresorhus** [comment]
Also FYI, you're using a lot of code from this project without complying with the MIT license.

**Comment by Hermanya** [comment]
I'm interested in finishing, if you are in merging.
How can I comply with the license?

**Comment by Hermanya** [comment]
I added `Copyright (c) Sindre Sorhus <sindresorhus@gmail.com> (sindresorhus.com)` to the license file. Let me know if I need to do anything else.

**Comment by sindresorhus** [comment]
> I added Copyright (c) Sindre Sorhus <sindresorhus@gmail.com> (sindresorhus.com) to the license file. Let me know if I need to do anything else.

That's correct :)

**Comment by sindresorhus** [comment]
> I'm interested in finishing, if you are in merging.

I like this, so 👍 from me. @bfred-it ?

**Comment by bfred-it** [comment]
There's still an issue in the code: It counts the visible followers, but there could be more over multiple pages.

Also the number style doesn't match GitHub and "Organizations" also lacks the number, so maybe we should just take the number off.

**Comment by Hermanya** [comment]
The did it the way it is on Twitter. But I think you are bringing up a good point.

**Comment by Hermanya** [comment]
Also, would it be okay, if I added https://github.com/hermanya/friendly-github to the "Related" section of the readme?

**Comment by sindresorhus** [comment]
This is very cool, @Hermanya. Thanks for contributing!

**Comment by sindresorhus** [comment]
> Also, would it be okay, if I added hermanya/friendly-github to the "Related" section of the readme?

Sure, do a PR ;)

**Pull request merged**